docs/post-mortem-jan-21.md

```diff
@@ -0,0 +1,246 @@
+
+# Actors severe mainnet bugs post-mortem – Jan 2021
+
+[**Intro**](#intro)
+
+[**Consensus Fault Author Check**](#A)
+	
+- [Part 1: Facts](#facts-A)
+- [Part 2: Discussion](#disc-A)
+
+[**Non-deterministic Map Iteration**](#B)
+
+- [Part 1: Facts](#facts-B)
+- [Part 2: Discussion](#disc-B)
+
+[**Multisig Infinite Re-entrancy**](#C)
+
+- [Part 1: Facts](#facts-C)
+- [Part 2: Discussion](#disc-C)
+
+[**Summary discussion and recommendations**](#recc)
+
+- [What *has* been working for us?](#working)
+
+
+
+# Intro
+<a name="intro"></a>
```

Comment:
Should be `id`, not name. Also, you _should_ be able to just link to `[Intro](#intro)` (i.e., `my-title-with-dashes-and-no-caps`).

Reply:
I tried the default heading names. But some of the subsections linked in table of contents had the same name which led me down this path.

Reply:
Ah... it looks like you can handle it by adding `-1`, `-2`, etc. But that's awkward.